omnibus/files/server-ctl-cookbooks/infra-server/attributes/default.rb

```diff
@@ -452,6 +452,36 @@
   '60' => '1000',
 }
 
```

Comment:
we should be able to get rid of most of the redis_lb entries - only hte stuff that runit cares about needs to remain in order to remove it (enable = false, dir, data_dir, log_*)